app/controllers/public/speaker/view.js

```diff
@@ -0,0 +1,9 @@
+import Controller from '@ember/controller';
+
+export default class extends Controller {
+
+  @computed('speaker.isAuthenticated', 'authManager.currentUser.email', 'authManager.currentUser.isAdmin', 'model.speakers')
```

Comment:
Codacy found an issue: ['computed' is not defined.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6403732)